app/controllers/products_controller.rb

```diff
@@ -0,0 +1,62 @@
+class ProductsController < ApplicationController
+  before_action :set_product, only: [:show, :edit, :update, :destroy]
+
+  def index
+    redirect_to root_path
+  end
+
+  # GET /products/new
```

Comment:
You can remove these comments, I think it's clear enough what is the method doing